components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java

```diff
@@ -296,15 +296,15 @@ private boolean insert(String columnNames, Object[] records, String tableName) t
             DBHandler.getInstance().insert(stmt);
             return true;
         } catch (SQLException e) {
-            throw new RDBMSTableException("Attempted execution of query [" + query + "] produced an exceptions" +
+            throw new RDBMSTableException("Attempted execution of insert query produced an exceptions" +
                     " in " + DATASOURCE_ID, e);
         } finally {
             if (stmt != null) {
                 try {
                     stmt.close();
                 } catch (SQLException e) {
                     //ignore
-                    logger.error("Error closing statement at inser.", e);
+                    logger.error("Error closing statement at insert.", e);
```

Comment:
Shall we add e.getMessage to the error?

Reply:
Fixed in acaafe89c43697be9ef1532a725f8c6bdc4ae6d6